doc/source/installation/windows_wsl.rst

```diff
@@ -328,6 +342,13 @@ Where ``$numprocs`` corresponds to the number of processors used for the compila
     
     apt-get numdiff
 
+  or
+
+  .. code-block:: text
+    :class: copy-button
+    
+    sudo apt install numdiff
```

Comment:
You don't need sudo privilege to any machine or user. I would keep it as a warning, but do not add it to the command itself. I suggest having a comment like "if you need superuser privilege, use sudo"